<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>importdeck</class>
 <widget class="QWidget" name="importdeck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1081</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="ImportDeckWidget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1921</width>
     <height>1081</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(38, 40, 47);</string>
   </property>
   <widget class="QLineEdit" name="EncoreDeckLinkField">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter the encore decks link here</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>10</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Import from encoredecks</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ImportButton">
    <property name="geometry">
     <rect>
      <x>858</x>
      <y>13</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeImportPanelButton">
    <property name="geometry">
     <rect>
      <x>1820</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="encoreDecksList">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>571</width>
      <height>841</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>100</y>
      <width>570</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Encore deck loaded list</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="encoreDecksList_2">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>130</y>
      <width>571</width>
      <height>841</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>100</y>
      <width>570</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Card list that will be loaded in the simulator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startImport">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>980</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
